src/app/Page.js

```diff
@@ -28,3 +72,58 @@ export default class Page extends React.Component<{|+children: Node|}> {
     );
   }
 }
+
+const style = StyleSheet.create({
+  header: {
+    color: "#1297A1",
+    fontFamily: '"Roboto Condensed", sans-serif',
+  },
+  body: {
+    padding: "0 100px",
+    flex: 3,
+  },
+  nav: {
+    height: 60,
+    padding: "20px 100px",
+  },
+  navItem: {
+    display: "inline-block",
+  },
+  navList: {
+    listStyle: "none",
+    paddingLeft: 0,
+    margin: 0,
+    display: "flex",
+  },
+  navLink: {
+    color: "#00848B",
+    fontFamily: "Roboto Condensed",
+    fontSize: 18,
+    textDecoration: "none",
+  },
+  navLinkHover: {
```

Comment:
Is there a reason that we would want to style a link with `navLink` but
not `navLinkHover`? If not, you can include the hover state as part of
the main link style:

```js
  navLink: {
    color: "#00848B",
    fontFamily: "Roboto Condensed",
    fontSize: 18,
    textDecoration: "none",
    ":hover": {
      color: "#08A27F",
      textDecoration: "underline",
    },
  },
```

Reply:
👍